File: sdkwork-app-sdk-csharp/Models/AchievementStatisticsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class AchievementStatisticsVO
    {
        public int? TotalAchievements { get; set; }
        public int? UnlockedAchievements { get; set; }
        public double? CompletionRate { get; set; }
        public int? TotalPoints { get; set; }
        public List<AchievementVO>? RecentAchievements { get; set; }
    }
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/AchievementStatisticsVO.kt
package com.sdkwork.app

data class AchievementStatisticsVO(
    val totalAchievements: Int? = null,
    val unlockedAchievements: Int? = null,
    val completionRate: Double? = null,
    val totalPoints: Int? = null,
    val recentAchievements: List<AchievementVO>? = null
)
